lib/kubeclient/common.rb

```diff
@@ -405,6 +405,15 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def process_template(template)
+      ns_prefix = build_namespace_prefix(template[:metadata][:namespace])
+      response = handle_exception do
+        rest_client[ns_prefix + 'processedtemplates']
+        .post(template.to_h.to_json, { 'Content-Type' => 'application/json' }.merge(@headers))
+      end
+      JSON.parse(response)
```

Comment:
No and I don't see rest_client has support for that. Our exception handler parses exceptions but other then that we parse the response manually on all calls but get_pod_log. I recently thought about wrapping some of the rest_client logic since it is repetitive. Maybe that belongs there too. Not in this PR though